testdroid_cmdline.sh

```diff
@@ -352,10 +358,11 @@ function get_result_files {
 #   test_run_id
 #   device_run_id
 # Returns:
-#   Void (writes files to subfolder 'results/')
+#   Void (writes files to subfolder TEST_RESULTS_DIR)
 #########################################
 function get_device_result_files {
-  mkdir -p "results"
+  rm -rf "${TEST_RESULTS_DIR:?}"
```

Comment:
This is theoretically not needed in a travis environment. However, if, for any reason it is not, there might be results files in that directory. This can cause adding failed results from previous runs.